systemtests/aci_test.go

```diff
@@ -15,39 +15,47 @@ func (s *systemtestSuite) TestACIMode(c *C) {
 	c.Assert(s.cli.GlobalPost(&client.Global{
 		Name:             "global",
 		NetworkInfraType: "aci",
-		Vlans:            "1-4094",
+		Vlans:            "1140-1150",
```

Comment:
This is a hw independent test that gets run as part of sanity. The vlan range should not matter for this test. However, the vlan range will matter for the other tests that use real ACI fabric. @gaurav-dalvi can we create a shared config file that will contain all the external fabric parameters: vlan range, tenant, physdom, external contracts etc. This can be a json/yml dict, keyed by the IP address of the setup. That way, each setup will gets its on vlan.